catalog/core/catalog-core-solr/src/main/java/ddf/catalog/source/solr/DynamicSchemaResolver.java

```diff
@@ -195,8 +198,11 @@ public DynamicSchemaResolver() {
    * SolrClient is up to ensure the cache is synchronized with Solr.
    *
    * @param client the SolrClient we are working with
+   * @throws SolrServerException if a Solr server exception occurs
+   * @throws SolrException if a Solr exception occurs
```

Comment:
Because it is a Runtime exception that can always be thrown by Solr. I added it as it seemed to have been forgotten all over the place when you look at the code. This makes it clear that it can be thrown out just as we would document IllegalArgumentException and the same ...